### PR TITLE
Add Gelato test dss-vest amounts

### PR DESCRIPTION
**Pull request by hexonaut**

### CODE REVIEW

File: src/Goerli-DssSpell.sol [hunk 1]
```diff
@@ -19,13 +19,23 @@ pragma solidity 0.6.12;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
+interface DssVestLike {
```
Comment (naszam) [inline src/Goerli-DssSpell.sol:22]: I think we don't need the DssVestLike interface as it's already included in [dss-interfaces](https://github.com/makerdao/dss-interfaces/blob/master/src/dss/VestAbstract.sol)

File: src/Goerli-DssSpell.t.sol [hunk 2]
```diff
@@ -42,6 +42,22 @@ interface PsmAbstract {
     function buyGem(address usr, uint256 gemAmt) external;
 }
 
+interface DssVestLike {
```
Comment (naszam) [inline src/Goerli-DssSpell.t.sol:45]: Same as above.

File: src/Goerli-DssSpell.sol (hunk 1 above)
Comment (hexonaut) [reply src/Goerli-DssSpell.sol:22]: That interface is out of date. `init` is now `create`, `usr()` is also missing. Just going to stick with the inline interfaces for now.

Comment (naszam) [reply src/Goerli-DssSpell.sol:22]: Oh! Good catch! 
We should fix that.

File: src/Goerli-DssSpell.sol [hunk 3]
```diff
@@ -19,13 +19,23 @@ pragma solidity 0.6.12;
...
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function file(bytes32, uint256) external;
```
Comment (naszam) [inline src/Goerli-DssSpell.sol:24]: I think we can remove it as not used in the spell.

File: src/Goerli-DssSpell.t.sol [hunk 4]
```diff
@@ -42,6 +42,22 @@ interface PsmAbstract {
...
+    function fin(uint256) external view returns (uint256);
+    function mgr(uint256) external view returns (address);
+    function res(uint256) external view returns (uint256);
+    function tot(uint256) external view returns (uint256);
+    function rxd(uint256) external view returns (uint256);
+    function unrestrict(uint256) external;
+    function vest(uint256) external;
+}
+
+
```
Comment (naszam) [inline src/Goerli-DssSpell.t.sol:60]: nit: extra line

**Review by naszam — APPROVED** [review]
LGTM :+1:

**Pull request merged**